.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do not change the file directory hierarchy. This will fail the workflow files and there is a special reason to follow that specific folder structure.

Reply:
Reverted